### PR TITLE
Support Python model predict for multiple outputs

### PR DESCRIPTION
**Pull request by hkvision**
Fix https://github.com/intel-analytics/zoo/issues/330

### CODE REVIEW

**Comment by hkvision** [comment]
cc @JasonChu1313

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/python/PythonZooKeras.scala [hunk 1]
```diff
@@ -125,6 +124,23 @@ class PythonZooKeras[T: ClassTag](implicit ev: TensorNumeric[T]) extends PythonB
     module.fit(trainData, epochs, valData)
   }
 
+  def zooPredict(
+      module: KerasNet[T],
+      x: JavaRDD[Sample],
+      batchSize: Int = 32): JavaRDD[JList[JTensor]] = {
+    val resRDD = module.predict(x.rdd.map(toJSample), batchSize)
+    resRDD.map(activityToJTensors).toJavaRDD()
```
[the remark this answers is not in the document]
Comment (zhichao-li) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/python/PythonZooKeras.scala]: good

**Comment by hkvision** [comment]
jenkins http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Validation/762/

**Comment by hkvision** [comment]
Thanks for your review!

**Pull request merged**